docs/src/extend/plugin-migration-flat-config.md

```diff
@@ -199,7 +199,39 @@ export default [
 ];
 ```
 
-You should update your documentation so your plugin users know how to reference the exported configs.
+If your config extends other configs, you can export an array:
+
+```js
+const baseConfig = require('./base-config');
+
+module.exports = {
+    configs: {
+        // the config referenced by example/recommended
+        extendedConfig: [
```

Comment:
If this config will be imported as `example.configs.recommended` it should be named `recommended`, not `extendedConfig`. Otherwise, it should be imported as `example.configs.extendedConfig` in the example file below, and in the legacy config system it would be referenced by `example/extendedConfig`.

```suggestion
        // the config referenced by example/recommended
        recommended: [
```

Reply:
It is already imported as `example.configs.extendedConfig`...?

Reply:
In that case, can we remove the comment that says `referenced by example/recommended` since that's a different config in this example?

Reply:
Aha good catch. Bad copy/paste.